src/SectorView.cpp

```diff
@@ -1018,16 +1030,7 @@ void SectorView::OnKeyPressed(SDL_Keysym *keysym)
 	if (Pi::IsConsoleActive())
 		return;
 
-	// space "locks" (or unlocks) the hyperspace target to the selected system
-	if (KeyBindings::mapLockHyperspaceTarget.Matches(keysym)) {
-		if ((m_matchTargetToSelection || m_hyperspaceTarget != m_selected) && !m_selected.IsSameSystem(m_current))
-			SetHyperspaceTarget(m_selected);
-		else
-			ResetHyperspaceTarget();
-		return;
-	}
-
-	if (KeyBindings::mapToggleSelectionFollowView.Matches(keysym)) {
+	if (InputBindings.mapToggleSelectionFollowView->Matches(keysym)) {
```

Comment:
So I'd actually much prefer attaching functions (or lambda functions) to the OnPressed callbacks of these input actions rather than the generic OnKeyPressed callback on Input. Among other reasons, this will work better with upcoming changes to the input framework, and allows the logic to be refactored a little more nicely.

Reply:
Done!